src/sqlancer/cnosdb/CnosDBExpectedError.java

```diff
@@ -0,0 +1,72 @@
+package sqlancer.cnosdb;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class CnosDBExpectedError {
+    private CnosDBExpectedError() {
+    }
+
+    public static List<String> expectedErrors() {
+        List<String> errors = new ArrayList<>();
```

Comment:
Here, it would be ideal to directly create an `ExpectedError` object rather than a list.